src/shadow-embed.js

```diff
@@ -548,6 +548,31 @@ export class ShadowDomWriterStreamer {
     return this.success_;
   }
 
+  /** @override */
+  get closed() {
+    return this.eof_;
+  }
+
+  /** @override */
+  get desiredSize() {
+    return 1; // noop
```

Comment:
Haha it evades you and me both 🦄. I *think* as long as it's positive, the exact size doesn't matter. The spec docs say:

> A producer can use this information to determine the right amount of data to write. 

It seems like the `desiredSize` value is a recommendation, which to me implies the exact value isn't important.